codecs/resize/src/lib.rs

```diff
@@ -48,5 +59,15 @@ pub fn resize(
     let mut output_image = Vec::<u8>::with_capacity(num_output_pixels * 4);
     output_image.resize(num_output_pixels * 4, 0);
     resizer.resize(input_image.as_slice(), output_image.as_mut_slice());
+
+    if premultiply {
+      for i in 0..num_output_pixels {
+        for j in 0..3 {
+          output_image[4 * i + j] = ((output_image[4 * i + j] as f32) * 255.0 / (output_image[4 * i + 3] as f32)) as u8;
```

Comment:
division by 0

Reply:
That was part of the reason why I chose `f32`, because then division by zero is not a panic and gets conveniently converted to `0` when cast to an `u8`.

[Playground example.](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=65e6c0d6621461fd15ac372a7dac8dfb)

Reply:
I should add a comment :D